web/src/components/tasks/business-formation/billing/BillingStep.test.tsx

```diff
@@ -54,6 +55,14 @@ describe("Formation - BillingStep", () => {
   const subtotalLabel = Config.formation.fields.paymentType.costSubtotalLabel;
   const totalLabel = "Total";
 
+  const composeSubTotalAria = (subTotalNumber: number): string => {
+    return `${subtotalLabel} ${getDollarValue(subTotalNumber)}`;
+  };
+
+  const composeTotalAria = (totalNumber: number): string => {
+    return `* ${totalLabel} ${getDollarValue(totalNumber)}`;
+  };
```

Comment:
[Non-Blocking] These look like virtually identical functions. Could we consolidate this?